_scripts/clone-legal-docs.sh

```diff
@@ -130,25 +135,32 @@ copy_md() {
 
     log "Copying $md_name and parent locale directories into $MODULE_PATH/public/legal-docs/"
 
+    cd "$DOWNLOAD_PATH"
     results=()
     for src in **"/${md_name}.md"; do
         [ -f "$src" ] || continue
-        legal_docs_dir="$MODULE_PATH/public/legal-docs/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
+        legal_docs_dir="$LOCAL_PATH/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
         [ -d "$legal_docs_dir" ] || mkdir -p "$legal_docs_dir"
         cp "$src" "$legal_docs_dir/$md_name.md"
         results+=("$(echo "$legal_docs_dir" | sed -E 's/.+\/([a-zA-Z_]+)/\1/; s/-/_/g')")
     done
 
-    log "Creating .json file containing array of available locales in $MODULE_PATH/public/legal-docs/${md_name}_locales.json"
+    cat "$LOCAL_PATH/stats.json" | jq ".\"${md_name}.md\".locales" > "$MODULE_PATH/public/legal-docs/${md_name}_locales.json"
```

Comment:
When I run `yarn storybook`, it errors out with this: `_scripts/clone-legal-docs.sh: line 148: jq: command not found`

Perhaps we need to use something else here? Oddly I didn't see the error when starting up (and I was able to test it fine locally) but I'm probably not digging through the logs enough.

Reply:
I did look into this earlier since I thought the `jq` requirement seemed odd (although you should 100% have it installed locally since it's an awesome tool).

Turns out we are using it in a few other places, but this might be the first time in _scripts/*-land (it seems to be in .circleci and .github/workflows previously):

```sh
git grep " jq " | cat

.circleci/create-lists.sh:    if [[ $(cat packages/$1/package.json | jq ".scripts.\"$2\"") != null ]]; then
.circleci/create-lists.sh:    if [[ $(cat packages/$1/package.json | jq ".scripts.\"$2\"") != null ]]; then
.github/workflows/l10n-gettext-extract.yml:            gh pr view $PR_NUMBER --json reviewRequests | jq -r ".reviewRequests[].name" | grep -q "fxa-l10n"
.github/workflows/l10n-gettext-extract.yml:                L10N_REVIEWERS=$(gh api -H "Accept: application/vnd.github+json" /orgs/mozilla/teams/fxa-l10n/members | jq -r '(.[].login | select(. != "clouserw"))' | sort)
.github/workflows/l10n-gettext-extract.yml:                PR_REVIEWERS=$(gh pr view $PR_NUMBER --json reviews | jq -r '.reviews[] | select(.state=="APPROVED") | .author.login' | sort)
_scripts/clone-legal-docs.sh:    cat "$LOCAL_PATH/stats.json" | jq ".\"${md_name}.md\".locales" > "$MODULE_PATH/public/legal-docs/${md_name}_locales.json"
```

If it's a new dev requirement, we might want to add a note somewhere in https://mozilla.github.io/ecosystem-platform/.